projects/angular/src/layout/nav/nav-level.ts

```diff
@@ -26,20 +26,23 @@ import { ClrStandaloneCdkTrapFocus } from '../../utils/focus/focus-trap';
 import { ResponsiveNavigationService } from './providers/responsive-navigation.service';
 import { ResponsiveNavCodes } from './responsive-nav-codes';
 
-import '@cds/core/internal-components/close-button/register.js';
-
-const createCdsCloseButton = (document: Document, ariaLabel: string) => {
-  const cdsCloseButton = document.createElement('cds-internal-close-button');
-  cdsCloseButton.setAttribute('icon-size', '32');
-  cdsCloseButton.setAttribute('aria-label', ariaLabel);
-  cdsCloseButton.setAttribute('aria-hidden', 'true');
-  cdsCloseButton.setAttribute('type', 'button');
+const createCloseButton = (document: Document, ariaLabel: string) => {
```

Comment:
I think this change should be extracted to a separate PR that just removes dependency on `cds-internal-close-button`.

Reply:
Yes, we can do it before the rest of this PR lands.